packages/frontend-shared/cypress/e2e/e2ePluginSetup.ts

```diff
@@ -126,6 +126,7 @@ async function makeE2ETasks () {
       await ctx.actions.app.ensureAppDataDirExists()
       await ctx.resetForTest()
       sinon.reset()
+      sinon.restore()
```

Comment:
just curious- doesn't `sinon.reset()` already reset the behavior of all the stubs (in addition to their state/history)? i'm not entirely sure why we need `sinon.restore()` here as well...

Reply:
reset will clean up standalone stubs, but stubbed methods on objects are only replaced with restore